### PR TITLE
[Python] Added tracking table for open issues & PRs related to concept exercises

### PR DESCRIPTION
**Pull request by BethanyG**
Feel free to prune if needed.   Added "tracking table" with Concept Name, Issue Link and PR Link columns to the reference README.md.

### CODE REVIEW

File: languages/python/reference/README.md [hunk 1]
```diff
@@ -538,3 +538,181 @@ These will very rarely be encountered in the wild, the first because it's more o
...
+| [Interfaces][integer-comparison] | [ ]() | [ ]() |
+| [Iterators][iterators] | [ ]() | [ ]() |
+| [Iterables][iterables] | [ ]() | [ ]() |
+| [Iteration][iteration] | [ ]() | [ ]() |
+| [is][keyword-is] | [ ]() | [ ]() |
+| [Keyword parameters][keyword-parameters]| [ ]() | [ ]() |
+| [Keyword-only parameters][keyword-only-parameters]| [ ]() | [ ]() |
+|[`**kwargs`][star-star-kwargs] | [ ]() | [ ]() |
+| [lambda][keyword-lambda] `intermediate`|   [ ]()         |         |
+| [list][builtin-types-list] `basic`|   [#859] (https://github.com/exercism/v3/issues/859)        |         |
```
Comment (yawpitch) [inline languages/python/reference/README.md:653]: Appears the space between the [label] and the (link) is screwing up rendering.

File: languages/python/reference/README.md [hunk 2]
```diff
@@ -538,3 +538,181 @@ These will very rarely be encountered in the wild, the first because it's more o
...
+| [Conditional structures][conditional-structures] | [ ]() | [ ]() |
+| [Constants][constants] | [ ]() | [ ]() |
+|[continue][keyword-continue]| [ ]() | [ ]() |
+| [Custom Classes][custom-classes] | [ ]() | [ ]() |
+| [decorators][decorators] `functional` | [ ]() | [ ]() |
+| [def][keyword-def] | [ ]() | [ ]() |
+| [Default arguments][default-arguments] | [ ]() | [ ]() |
+|[del][keyword-del] |  [ ]() | [ ]() |
+| [dict][builtin-types-dict] `basic`|   [#877](https://github.com/exercism/v3/issues/877)  |         |
+| [Dict comprehension][dict-comprehension] |  [ ]() | [ ]() |
```
Comment (yawpitch) [inline languages/python/reference/README.md:592]: There's a link reference for `dict-comprehension` here, but there's no corresponding link entry ... we'll need to add `[dict-comprehension]: ./concepts/dict_comprehension.md` in the appropriate location in the link entries.

File: languages/python/reference/README.md [hunk 3]
```diff
@@ -538,3 +538,181 @@ These will very rarely be encountered in the wild, the first because it's more o
...
+| [Iterators][iterators] | [ ]() | [ ]() |
+| [Iterables][iterables] | [ ]() | [ ]() |
+| [Iteration][iteration] | [ ]() | [ ]() |
+| [is][keyword-is] | [ ]() | [ ]() |
+| [Keyword parameters][keyword-parameters]| [ ]() | [ ]() |
+| [Keyword-only parameters][keyword-only-parameters]| [ ]() | [ ]() |
+|[`**kwargs`][star-star-kwargs] | [ ]() | [ ]() |
+| [lambda][keyword-lambda] `intermediate`|   [ ]()         |         |
+| [list][builtin-types-list] `basic`|   [#859] (https://github.com/exercism/v3/issues/859)        |         |
+| [list comprehension][list-comprehension] | [ ]() | [ ]() |
```
Comment (yawpitch) [inline languages/python/reference/README.md:654]: There's a link reference for `list-comprehension` here, but there's no corresponding link entry ... we'll need to add `[list-comprehension]: ./concepts/list_comprehension.md` in the appropriate location in the link entries.

File: languages/python/reference/README.md [hunk 4]
```diff
@@ -538,3 +538,181 @@ These will very rarely be encountered in the wild, the first because it's more o
...
+| [String splitting][string-splitting] `basic`| [ ]() | [ ]() |
+| [String translation][string-translation] `basic`| [ ]() | [ ]() |
+| [True][keyword-true]| [ ]() | [ ]() |
+| [try][keyword-try]| [ ]() | [ ]() |
+|[tuple][builtin-types-tuple] `basic`| [ ]() | [ ]() |
+|[Tuple unpacking][tuple-unpacking]| [ ]() | [ ]() |
+| [type][builtin-types-type] `OOP`| [ ]() | [ ]() |
+| [type conversion][type-conversion] `basic`| [ ]() | [ ]() |
+| [Type hinting][type-hinting] | [ ]() | [ ]() |
+| [ [Variables]] [variables] | [ ]() | [ ]() |
```
Comment (yawpitch) [inline languages/python/reference/README.md:714]: Space between [label] and [reference] is breaking rendering.

File: languages/python/reference/README.md (hunk 2 above)
Comment (BethanyG) [reply languages/python/reference/README.md:592]: Added in entries for dict, list and set.  😄

File: languages/python/reference/README.md (hunk 1 above)
Comment (BethanyG) [reply languages/python/reference/README.md:653]: Added a `Section` column, and put in the sections.  This probably needs editing, and maybe linking...ooof.

I both love and hate markdown.  Beats HTML...**BUT**  

I corrected all the links I could find, but given the length of this table, chances are good I munged something else....

File: languages/python/reference/README.md (hunk 4 above)
Comment (BethanyG) [reply languages/python/reference/README.md:714]: Clearly, I need another markdown editor (I am only half-joking here).  PyCharm, WebStorm, Macdown, AND Typora have all failed me in one way or another.  So have the plugins for Atom.  Never thought I'd miss Excel (and I don't _really_) .. but editing a long table sure is handy in spreadsheet form.

**Comment by cmccandless** [comment]
This is great! Thanks @BethanyG!

Related (but not redundant): https://github.com/exercism/v3/projects/13

**Pull request merged**